Tasks/AzureIoTEdgeV2/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
```

Comment:
>version [](start = 5, length = 7)

what about the version in the package.json, do we need to increment it as well?

Reply:
@Josh-01 do we need to update version numbers in package.json when we do updates to an extension?

Reply:
Just the task.json version update should do.